content/en/docs/reference/node/systemd-watchdog.md

```diff
@@ -29,7 +30,7 @@ WatchdogSec=30s
 ```
 
 Setting `WatchdogSec=30s` indicates a service watchdog timeout of 30 seconds.
-Within the kubelet, the `sd_notify()` function is invoked, at intervals of `WatchdogSec` ÷ 2, to send
+Within the kubelet, the `sd_notify()` function is invoked, at intervals of \\( WatchdogSec \div 2 )). to send
```

Comment:
This doesn't render correctly for me, currently. Based on [this documentation](https://www.docsy.dev/docs/adding-content/diagrams-and-formulae/#inline-formulae), I believe the notation needs changing?
```suggestion
Within the kubelet, the `sd_notify()` function is invoked, at intervals of \\( WatchdogSec \div 2\\). to send
```

Reply:
D'oh. I thought I'd checked this one.